frontend/app/src/people/utils/nameTag.tsx

```diff
@@ -52,51 +111,63 @@ export default function NameTag(props) {
       }}
       style={style}>
       {!isSelected && (
-        <>
-          <Img src={img || `/static/person_placeholder.png`} iconSize={iconSize} />
-          <Name textSize={textSize}>{owner_alias}</Name>
-
+        <div
+          style={{
+            display: 'flex',
+            flexDirection: 'row'
+          }}>
+          <Img src={img || `/static/person_placeholder.png`} iconSize={32} isPaid={isPaid} />
           <div
             style={{
-              height: 3,
-              width: 3,
-              borderRadius: '50%',
-              margin: '0 6px',
-              background: '#8E969C'
-            }}
-          />
-        </>
-      )}
-
-      <Date>{lastSeen}</Date>
-      {ticketUrl && (
-        <GithubIcon
-          onClick={(e) => {
-            e.stopPropagation();
-            window.open(ticketUrl, '_blank');
-          }}>
-          <img height={'100%'} width={'100%'} src="/static/github_logo.png" alt="github" />
-        </GithubIcon>
-      )}
-      {loomEmbedUrl && (
-        <LoomIcon
-          onClick={(e) => {
-            e.stopPropagation();
-            window.open(loomEmbedUrl, '_blank');
-          }}>
-          <img height={'100%'} width={'100%'} src="/static/loom.png" alt="loom" />
-        </LoomIcon>
+              display: 'flex',
+              flexDirection: 'column',
+              marginLeft: '14px'
+            }}>
+            <Name textSize={textSize} color={isPaid ? '#B0B7BC' : '#000'}>
```

Comment:
Same thing with the color
https://github.com/stakwork/sphinx-tribes/blob/master/frontend/app/src/colors.tsx